pkg/virtctl/root_test.go

```diff
@@ -0,0 +1,162 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2018 Red Hat, Inc.
+ *
+ */
+
+package virtctl
+
+import (
+	"io/ioutil"
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+
+	"kubevirt.io/kubevirt/pkg/kubecli"
+)
+
+var exampleYaml = `name: virt
+shortdesc: kubevirt command plugin
```

Comment:
This should be generated. You can probably take a similar approach like done here: https://github.com/spf13/cobra/blob/master/bash_completions.go (but without templates since we have a struct which we want to fill).

Reply:
Sorry my fault. For whatever reason I thought that this is the yaml which we will write to disk. Everything good here.